.travis.yml

```diff
@@ -0,0 +1,21 @@
+sudo: required
+
+language: java
+
+services:
+  - docker
+
+before_install:
+  - wget http://mirrors.ae-online.de/apache/maven/maven-3/3.3.9/binaries/apache-maven-3.3.9-bin.tar.gz
+  - tar xf apache-maven-3.3.9-bin.tar.gz
+  - export M2_HOME=$PWD/apache-maven-3.3.9
+  - export PATH=$M2_HOME/bin:$PATH
```

Comment:
@bartoszmajsak we need maven >= 3.2 which is not there in travis. We have maven 2.x installed on travis. So it's needed.